week1/Q2.py

```diff
@@ -0,0 +1,51 @@
+class Node:
+	def __init__(self, value = None, next = None):
+		self.value = value
+		self.next = next
+
+
+class LinkedList:
+	def __init__(self):
+		self.first = None
+		self.last = None
+		self.length = 0
+
+	def clear(self):
+		self.__init__()
+
+	def add(self, x):
+		self.length += 1
+		if self.first == None:
+			self.last = self.first = Node(x, None)
+		else:
+			self.last.next = self.last = Node(x, None)
+
+
+def get_kth_to_last(L, k):
```

Comment:
Usually arguments in Python are named with lowercase variables.
You can use, instead of `L`, something like `list`.